fflas-ffpack/ffpack/ffpack.h

```diff
@@ -1198,6 +1202,13 @@ namespace FFPACK { /* Solutions */
          size_t* P, size_t* Q,
          const FFLAS::FFLAS_DIAG Diag=FFLAS::FflasNonUnit);
 
+    template <class Field, class PSHelper>
+    typename Field::Element&
+    Det( typename Field::Element& det,
```

Comment:
There should be no Det in this PR. Please remove this change

Reply:
I saw that some modifications had been added into the master branch then I merged this branch with the recent version of master branch and I see no big difference so I wonder if it is still necessary to revert this Det function.